.github/config.yml

```diff
@@ -24,3 +24,43 @@ requestInfoDefaultTitles:
 
 # *OPTIONAL* Label to be added to Issues and Pull Requests with insufficient information given
 requestInfoLabelToAdd: needs-more-info
+
+###
+
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue in this project! Your input is very important to us! :hugs:
+
+  <br>If you haven't done so already, check out our [Code of Conduct](https://github.com/jupyter/governance/blob/master/conduct/code_of_conduct.md).
```

Comment:
"our" is not obvious what it refers to, is it this project or Jupyter's? Should we say Jupyter's Code of Conduct explicitly?